python-backend/tests/constants.py

```diff
@@ -55,8 +55,8 @@
 TEST_REGION_GUID = 'a0b98cb1-6e70-4c54-9395-1e8ccc867cd1'
 TEST_REGION_CODES = ['SW','SC','NW','NE','SE']
 TEST_REGION_CODE_DISPLAY_ORDER = [10,20,30,40,50]
-TEST_REGION_CODE_1 = 'SW'
-TEST_REGION_DESCRIPTION= 'TEST CODE'
+TEST_REGION_CODE = 'NE'
+TEST_REGION_DESCRIPTION= 'North East Region'
```

Comment:
Do we still have a use for descriptions on the server? I guess I misunderstood, but I thought they were being removed. Or maybe they were just being returned to the client in a different way?

Reply:
We still have description in the mine_region_code table. But we removed it in one of the returned lists.